### PR TITLE
Back test

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]
ok for me

File: frontend/src/services/context/Ctx.jsx [hunk 1]
```diff
@@ -1,92 +1,19 @@
...
-    },
-    {
-      name: "name-13",
-      role: "role-13",
-      number: 13,
-      img: "./src/assets/images/cards/players/photo.png",
-    },
-  ];
+  useEffect(() => {
+    axios.get("http://localhost:5000/players").then(({ data }) => {
```
Comment (LuigiMenez) [inline frontend/src/services/context/Ctx.jsx:13]: Use environement variable for call of axios

**Review by LuigiMenez — APPROVED** [review]
I have not been able to see two files to ensure that they are not corrupted

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
use variable environement, I have not been able to see two files to ensure that they are not corrupted

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok, be careful with the branch name, not very clear

**Pull request merged**